telegram/_message.py

```diff
@@ -194,19 +198,19 @@ class Message(TelegramObject):
             optional): Service message: auto-delete timer settings changed in the chat.
 
             .. versionadded:: 13.4
-        migrate_to_chat_id (:obj:`int`, optional): The group has been migrated to a supergroup with
-            the specified identifier. This number may be greater than 32 bits and some programming
-            languages may have difficulty/silent defects in interpreting it. But it is smaller than
-            52 bits, so a signed 64 bit integer or double-precision float type are safe for storing
-            this identifier.
+        migrate_to_chat_id (:obj:`int`, optional): The group has been migrated to a supergroup
+            with the specified identifier. This number may be greater than 32 bits and some
+            programming languages may have difficulty/silent defects in interpreting it. But it is
+            smaller than 52 bits, so a signed 64 bit integer or double-precision float type are
+            safe for storing this identifier.
```

Comment:
```suggestion
            with the specified identifier.
```
We can remove this since this doesn't apply to us. Same in the Attributes section

Reply:
done